smartseq2_single_sample/ss2_single_sample.wdl

```diff
@@ -4,7 +4,7 @@ import "hisat2_rsem_pipeline.wdl" as run_hisat2_rsem
 ## QC pipeline: HISAT2+Picard on genome alignments
 ## quantification pipeline: HISAT2+RSEM on transcriptome alignments
 
-workflow run_pipeline {
+workflow Ss2RsemSingleSample {
```

Comment:
I like the change, but could you include Hisat2 in the name as well to be more descriptive, and since that has changed since the last iteration? @dshiga Would my suggestion be consistent with your request?

Reply:
Now that you mention it, do we really need Rsem in there? Why not just Ss2SingleSample or Ss2 for that matter? Do we expect to have multiple ss2 pipelines, some using rsem and hisat, and others using other aligners?

Reply:
My preference if I could name the production pipeline, would be for `SmartSeq2`as I think that most accurately encapsulates the data type we expect to process with this pipeline. 

There is the trickiness that technically this pipeline only runs on one cell and the _adapter_ scatters over the cells. For this reason I would also accept `OneCellSmartSeq2` or similar. I don't think `SingleSample` conveys this information, as many might think that experiment equates to sample. Similarly, I think `SingleCell` is overloaded in this context. One would naturally assume we refer to single-cell sequencing.

Reply:
... corollary: If we go with `SmartSeq2`, we should put a note into the workflow about dependence upon the adapter to process a complete multi-cell experiment.

Reply:
I think we should use a name not too specific but also descriptive, I would call it something like `Ss2Hisat2RsemPipeline` and a wrapper which scatter-gather this pipeline would be called something like `Ss2Hisat2RsemPipelineWrapper`